jenkins/README.md

```diff
@@ -89,16 +148,9 @@ found [here](https://github.com/Nordix/airship-dev-tools/blob/master/ci/images/g
 pre-baked Kubernetes executables. The image building script for CentOS can be
 found [here](https://github.com/Nordix/airship-dev-tools/blob/master/ci/images/gen_metal3_centos_image.sh).
 
-## Job logs
```

Comment:
This section is removed. Is this not true anymore?

Reply:
It is, we just moved it a bit up https://github.com/Nordix/metal3-project-infra/tree/add-jenkins-instructions/feruz/jenkins#job-logs

Reply:
It is, it just moved a bit up https://github.com/Nordix/metal3-project-infra/tree/add-jenkins-instructions/feruz/jenkins#job-logs